opensciencegrid/gracc-apel/Dockerfile

```
@@ -0,0 +1,28 @@
+ARG BASE_OSG_SERIES=3.5
+
+FROM hub.opensciencegrid.org/opensciencegrid/software-base:$BASE_OSG_SERIES-el7-release
+
+# install dependencies
+RUN yum -y install python3 python3-pip && \
+  pip3 install elasticsearch-dsl && \
+  pip3 install argo-ams-library
+RUN yum -y install http://rpm-repo.argo.grnet.gr/ARGO/devel/centos7/python-argo-ams-library-0.5.5-20210415071520.ff0c536.el7.noarch.rpm
+
+RUN yum -y install python-dirq
+
+# install ca-policy-egi-core
+RUN yum -y remove osg-ca-certs
+RUN cd /etc/yum.repos.d/ && \
+  wget http://repository.egi.eu/sw/production/cas/1/current/repo-files/EGI-trustanchors.repo && \
```

```suggestion
RUN curl -so /etc/yum.repos.d/EGI-trustanchors.repo https://repository.egi.eu/sw/production/cas/1/current/repo-files/EGI-trustanchors.repo && \
```

`wget` isn't in `software-base`; also use HTTPS

---

@yongleyuan what's the status on this one? It's been nearly 3 weeks!